internal/loader/configuration_setting_loader.go

```diff
@@ -46,6 +46,8 @@ type TargetKeyValueSettings struct {
 	// Multiple secrets could be managed
 	SecretSettings   map[string]corev1.Secret
 	SecretReferences map[string]*TargetSecretReference
+	KeyValueETags    map[acpv1.Selector][]*azcore.ETag
```

Comment:
Can we avoid using both `KeyValueEtags` and `FeatureFlagETags`, can unify them into `PageEtags` ?

Reply:
Ok, we can't since it may return them both